Removed kwargs from node.run_sstablesplit

pauloricardomg opened the pull request:
[PR363](https://github.com/pcmanus/ccm/pull/363) caused a [regression](http://cassci.datastax.com/view/cassandra-3.0/job/cassandra-3.0_dtest/lastCompletedBuild/testReport/sstablesplit_test/TestSSTableSplit/single_file_split_test/history/) on dtest `single_file_split_test` because that test specifies the `stdout` paramter on `kwargs`, what causes a conflict when calling `subprocess.Popen`, with `stdout=subprocess.PIPE`.

The fix is to provide and `output_file` parameter instead, similar to what is done in other tool methods, such as `run_sstablemetadata` or `run_sstableexpiredblockers`, so we can make a conditional on that before calling `subprocess.Popen`.

pauloricardomg commented:
The cassandra-dtest PR for using the new option on `single_file_split_test` is available [here](https://github.com/riptano/cassandra-dtest/pull/497).

mambocab commented:
@ptnapoleon has the final say on this, but I would much prefer this return a string than work directly with file handles. I know that that's how some of the existing methods work, but I don't like that behavior either :P

pauloricardomg commented:
@mambocab agreed.  `node.run_sstablesplit` already returns the standard output and error as string, so I modified the test to use that output instead of passing the `output_file` parameter.

Updated the PR to remove `kwargs`  from `run_sstablesplit`, since it is no longer necessary.

ptnapoleon commented:
+1
